code/game/objects/items/ego_weapons/non_abnormality/kcorp.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/ego_weapon/city/kcorp
+	name = "K corp baton"
+	desc = "A green baton used by K corp employees."
+	icon_state = "kbatong"
+	inhand_icon_state = "kbatong"
+	force = 22
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("bashes", "crushes")
+	attack_verb_simple = list("bash", "crush")
+
+//Slows you to half but has really good defenses
+/obj/item/ego_weapon/shield/kcorp
+	name = "K corp riot shield"
+	desc = "A riot shield used by K corp employees."
+	special = "Slows down the user significantly."
+	icon = 'ModularTegustation/Teguicons/lc13_weapons.dmi'
+	icon_state = "kshield"
+	inhand_icon_state = "flashshield"
+	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
+	force = 15
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("shoves", "bashes")
+	attack_verb_simple = list("shove", "bash")
+	hitsound = 'sound/weapons/genhit2.ogg'
+	reductions = list(70, 70, 70, 20)
+	recovery_time = 5 SECONDS
+	block_time = 2 SECONDS
+	block_recovery = 3 SECONDS
```

Comment:
Still no requirements for a shield that contends with our non-implemented WAW shields and likely surpasses them easily. I'd suggest a max stat line of 60/60/60/30 for reductions due to the sheer LENGTH of the block meaning unless something has an irregularly slow attack speed or an erratic one, it's nigh impossible to fuck it up.